src/Layout/AppLayout.jsx

```diff
@@ -168,7 +172,8 @@ const Sidebar = styled.div(
           borderRightColor: 'lighterOutline'
         }
       }
-    })
+    }),
+  ({ sidebarContentCss }) => css({ ...sidebarContentCss })
```

Comment:
Although I don't think it will break things if the variable is undefined, maybe it's better to have an empty object as default value in `defaultProps` for the `AppLayout`.

Reply:
a680521